plugins/node/opentelemetry-instrumentation-graphql/src/index.ts

```diff
@@ -15,4 +15,4 @@
  */
 
 export * from './instrumentation';
-export { GraphQLInstrumentationConfig } from './types';
+export * from './types';
```

Comment:
generally LGTM, but the `types.ts` file exports many many internal stuff which are not useful to the instrumentation users.
WDYT about additionally exporting just the `GraphQLInstrumentationExecutionResponseHook` and not everything?

Reply:
Fixed, Thanks!